### PR TITLE
Fix tempest errors

### PR DESCRIPTION
**Pull request by imkarrer**
Tempest currently runs for about 1.5 - 2 hours.  I have included a copy of the last output and commented modifications to Jumpgate which allow Tempest get deeper into the tests.  
1) It skipped one but was able to run 63 with 2 errors and 8 failures.  I am not sure which tests were the ones that passed.  There are ~75 tests to run in the whitelist.
Ran 64 tests in 5047.007s

FAILED (SKIP=1, errors=2, failures=8)

2) cancel_jumpgate_instance.sh is the script which cancels jumpgate instances and volumes created during the tempest tests.  You may need to edit the regex to grep against.  Right now the script uses SoftLayer CLI and deletes all instances which end with jumpgate.com.

### CODE REVIEW

**Comment by imkarrer** [comment]
After issuing the pull request I realized that  unit tests need modifications to pass with the current changes to jumpgate.  Closing temporarily.  Will reopen once all issues are resolved.

File: jumpgate/compute/drivers/sl/servers.py [hunk 1]
```diff
@@ -593,13 +608,12 @@ def get_server_details_dict(app, req, instance, is_list):
         'tenant_id': tenant_id,
         # NOTE(bodenr): userRecordId accessibility determined by permissions
         # of API caller's user id and api key. Otherwise it will be None
-        'user_id': utils.lookup(instance,
-                                'billingItem',
-                                'orderItem',
-                                'order',
-                                'userRecordId'),
+        # TODO(imkarrer) - returning None makes tempest fail, conditionally returning empty string  # noqa
```
Comment (jimlindeman) [inline jumpgate/compute/drivers/sl/servers.py]: I see several lines like this saying T0D0 but not really showing anything to change in that part of the code in the future.  Should it just be a plain comment line:
# returning None makes tempest fail, conditionally returning empty string

File: tempest_output.txt [hunk 2]
```diff
@@ -0,0 +1,303 @@
+F........F..FF.E......S........FFFF....................E.........
```
Comment (jimlindeman) [inline tempest_output.txt]: Is this file really intended to be checked in?

Comment (imkarrer) [inline tempest_output.txt]: I included it so that who ever runs tempest next does not need to wait 1.5 hours for the output.  I will take it out of the commit if it has no business here.

File: jumpgate/compute/drivers/sl/servers.py (hunk 1 above)
Comment (imkarrer) [inline jumpgate/compute/drivers/sl/servers.py]: I see. I will make the changes accordingly.  Thank you.

**Comment by jimlindeman** [comment]
+1
Looks good
